### PR TITLE
Install libs/cmake/pkgconfig thingy into right place for MinGW

### PR DESCRIPTION
**Pull request by horar**
MinGW folder structure is closer to Linux.
This allow use other MinGW package to utilize CMake or pkgconfig
to setup the project.

With these changes, installed stuff go into:
-- Install configuration: "Release"
-- Installing: /usr/i686-w64-mingw32/sys-root/mingw/lib/modules/config_kde.dll
-- Installing: /usr/i686-w64-mingw32/sys-root/mingw/lib/pkgconfig/libproxy-1.0.pc
-- Installing: /usr/i686-w64-mingw32/sys-root/mingw/share/cmake/Modules/Findlibproxy.cmake
-- Installing: /usr/i686-w64-mingw32/sys-root/mingw/lib/liblibproxy.dll.a
-- Installing: /usr/i686-w64-mingw32/sys-root/mingw/lib/libproxy.dll
-- Up-to-date: /usr/i686-w64-mingw32/sys-root/mingw/include/proxy.h
-- Installing: /usr/i686-w64-mingw32/sys-root/mingw/bin/proxy.exe

It add stuff into 'lib' folder and finally other project was able to find the library.

### CODE REVIEW

File: libproxy/cmake/devfiles.cmk [hunk 1]
```diff
@@ -1,4 +1,4 @@
-if(NOT WIN32)
+if(NOT WIN32 OR MINGW)
```
Comment (DimStar77) [inline libproxy/cmake/devfiles.cmk]: For completeness the related endif should also contain the condition; can you please fix that?

Comment (horar) [inline libproxy/cmake/devfiles.cmk]: It is necessary for old CMake only, but OK; done :-)

Comment (DimStar77) [inline libproxy/cmake/devfiles.cmk]: Can you squash the commits into one?

Comment (horar) [inline libproxy/cmake/devfiles.cmk]: I don't see this option here, probably only when I trigger new pull-request.
But ghub should support it on you side ... if enabled, you should see squash&merge

**Pull request merged**